community/ctags/sources

```diff
@@ -0,0 +1 @@
+http://prdownloads.sourceforge.net/ctags/ctags-5.8.tar.gz
```

Comment:
This appears to be the old unmaintained version of ctags; I think other distros tend to package ctags from the maintained fork at https://github.com/universal-ctags/ctags

Reply:
So what I submitted was Exuberant Ctags and you linked the newer Universal Ctags. I went with the older version because I'm personally not a fan of ./autogen.sh which is part of the build process of the latter. Outside of this preference, there are other differences which might justify having both. If most people consider ctags to be the Universal flavor, can give up the name and move this package to e-ctags.